.github/workflows/labeler.yml

```
@@ -0,0 +1,12 @@
+name: Label pull request
+on:
+  - pull_request_target
```

```suggestion
on:
  pull_request_target:
    branches:
      - master
    types:
      - opened
```
dont need other branches and also i think i would find this annoying if it added back labels that I intentionally removed, so just `opened`